dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/columns/image-preview.test.js

```diff
@@ -46,5 +46,39 @@ define([
             });
 
         });
+
+        describe('handleKeyDown method', function () {
+            it('veify record changed on key down', function () {
+                var recordMock = {
+                    _rowIndex: 2
+                },
```

Comment:
@Nazar65 @engcom-Golf the formatting is not consistent, lines 53-54 require additional indentation

Reply:
ok